src/test/java/com/springbootWithJPA/springBootRestExample/StepDefinitions.java

```diff
@@ -0,0 +1,86 @@
+package com.springbootWithJPA.springBootRestExample;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.junit.Assert;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.*;
+
+public class StepDefinitions {
+    @Autowired
+    SpringBootRestExampleIntegrationTest springBootRestExampleIntegrationTest;
+    Employee employee = new Employee();
+    Employee emp;
+    Employee[] employees;
+
+    @Given("Valid employee object")
+    public void valid_employee_object() {
+        employee.setFirstName("Soumya");
+        employee.setLastName("Nagesh");
+        employee.setEmailId("Soumya@gmail.com");
+    }
+
+    @When("I  send a  request a new employee is created in db")
+    public void send_a_request_with_Id_firstName_Jashika_lastName_Vasu_emailId_JashikaVasu_gmail_com_a_new_employee_is_created() {
```

Comment:
This should not be such a long method name. Also use camelcase naming conventions like
sendRequest
createRequest etc

Reply:
addressed in https://github.com/Jashika/springBootRestExample/pull/2/commits/9e9a7aac750f7c921ed6227482cdfeaee316c12a